src/Modules/LIKQ/FanoutSearch/Descriptors/NodeDescriptor.cs

```diff
@@ -86,7 +86,7 @@ private string EncodeValue(string v)
             if (v == string.Empty) return "\"\"";
             var fchar = v[0];
 
-            if (char.IsLetter(fchar)) // probably a string. don't parse
+            if (char.IsLetter(fchar) || char.Equals('0', fchar)) // probably a string. don't parse
```

Comment:
why 0? could you give an example?

Reply:
I defined a cell:
```
cell MyCell
{
string Code;
}
```

Then I have a query with LambdaDSL like:
```
MAG.StartFrom("target cell id", new[]{"Code"})
```
If the Code value is '00001' the result will be:
```
// code part of the result
{
  "Code": 00001
}
```
This is not a valid json string, parser will complain about it(i am using python here), But if I use'*' instead of 'Code', it will return a string value under '*'.

Reply:
Yes, Code="1000" will return a number, but we can still get a valid json result, so i can still use it as i do know the type.
Then I did not have a schema for json parse, as schema can only validate a deserialized object not a json string.

Reply:
ooh you mean `0001` is invalid json because of the prefix 0s? that's more interesting now...

Reply:
yes, but json.net consider it as a number, confusing..